tests/integration/landingai/test_predict_e2e.py

```diff
@@ -67,9 +67,9 @@ def test_od_predict():
     assert len(preds) == 3, "Result should not be empty or None"
     expected_scores = [0.9997851252555847, 0.9983770251274109, 0.9983124732971191]
     expected_bboxes = [
-        (432, 1036, 652, 1203),
-        (948, 1592, 1122, 1798),
-        (1518, 1413, 1991, 1799),
+        (432, 1036, 653, 1204),
+        (948, 1592, 1123, 1799),
+        (1518, 1413, 1992, 1800),
```

Comment:
@Dayof recently changed the way in which we cast float bboxes to int bboxes and that can result in 1 pixel changes in the values. This is a reflection of that.